src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -69,20 +69,20 @@ Regenerate the directory by running `flutter clean` or `flutter pub get` in the
 
 There are two ways to embed Flutter in your existing application.
 
-1. Use the CocoaPods dependency manager and installed Flutter SDK. Recommended.
-1. Create frameworks for the Flutter engine, your compiled Dart code, and all
-   Flutter plugins. Manually embed the frameworks and update your existing
+1. Use the CocoaPods dependency manager and installed Flutter SDK. (Recommended.)
+2. Create frameworks for the Flutter engine, your compiled Dart code, and all
```

Comment:
Having a list of ordered lists by just prefacing 1. actually makes markdown show the right numbers sequentially without having to explicitly track the numbers in the source text.

Reply:
OK. Thanks!